Makefile

```diff
@@ -95,7 +95,7 @@ docker_run:
 		-e GOOGLE_APPLICATION_CREDENTIALS=${CREDENTIALS_PATH} \
 		-v $(CURDIR):/cft/workdir \
 		${DOCKER_REPO_BASE_KITCHEN_TERRAFORM} \
-		/bin/bash -c "source test/ci_integration.sh && setup_environment && exec /bin/bash"
+		/bin/bash -c "cd /cft/workdir && source test/ci_integration.sh && setup_environment && exec /bin/bash"
```

Comment:
What was the motivation for this change?

Reply:
It related to changes in the behavior of `kitchen-terraform:2.4.0`, please take a look:
weannye@hrk1-lhp-f73386:~/work/gcp/forks/terraform-google-vm$ make docker_run
docker run --rm -it \
       -e PROJECT_ID \
       -e SERVICE_ACCOUNT_JSON \
       -e GOOGLE_APPLICATION_CREDENTIALS=/cft/workdir/credentials.json \
       -v /home/weannye/work/gcp/forks/terraform-google-vm:/cft/workdir \
       gcr.io/cloud-foundation-cicd/cft/kitchen-terraform:2.4.0 \
       /bin/bash -c "source /cft/workdir/test/ci_integration.sh && setup_environment && exec /bin/bash"
Activated service account credentials for: [project-factory-21468@gl-ipukha.iam.gserviceaccount.com]
/cft/workdir/test/ci_integration.sh: line 35: test/fixtures/shared/credentials.json: No such file or directory
bash-5.0# pwd
/workspace
bash-5.0# exit

So as you see, it can't work from the wrong directory.